roles/cloud-cloudstack/tasks/main.yml

```diff
@@ -0,0 +1,71 @@
+---
+- block:
+    - name: Build python virtual environment
+      import_tasks: venv.yml
+
+    - block:
+      - name: Include prompts
+        import_tasks: prompts.yml
+        
+      - set_fact:
+          algo_region: >-
+            {% if region is defined %}{{ region }}
+            {%- elif _algo_region.user_input is defined and _algo_region.user_input != "" %}{{ cs_zones[_algo_region.user_input | int -1 ]['name'] }}
+            {%- else %}{{ cs_zones[default_region | int - 1]['name'] }}{% endif %}
```

Comment:
I tried to deploy with the default values and got this error:
> The error was: 'default_region' is undefined

Reply:
fixed, typo. sorry